### DISCUSSION TITLE
separating content from code

### DISCUSSION THREAD

**Discussion by nabajour** [getnikola/nikola#3642] [Q&A] [answered]
Hi, 

I like to have my website content separated from the code creating the website (I already have a bunch of orgmode notes in another git repository I would like to transform into posts). I tried moving the nikola code to `/home/me/nikola` and the content to `/home/me/website_content/`. I moved the  `pages`, `posts`, `files`, `folders`, `images`, `listings`, `galleries` into that folder. 

I managed to get it mostly to work with change along these lines in `conf.py`:

```
CONTENT_HOME = "/home/me/website/"
 POSTS = (
    (CONTENT_HOME + "posts/*.org", "blog", "post.tmpl"),
    (CONTENT_HOME + "posts/*.rst", "blog", "post.tmpl"),
    (CONTENT_HOME + "posts/*.md", "blog", "post.tmpl"),
    (CONTENT_HOME + "posts/*.txt", "blog", "post.tmpl"),
    (CONTENT_HOME + "posts/*.html", "blog", "post.tmpl"),
 )
 PAGES = (
    (CONTENT_HOME + "pages/*.org", "", "page.tmpl"),
    (CONTENT_HOME + "pages/*.rst", "", "page.tmpl"),
    (CONTENT_HOME + "pages/*.md", "", "page.tmpl"),
    (CONTENT_HOME + "pages/*.txt", "", "page.tmpl"),
    (CONTENT_HOME + "pages/*.html", "", "page.tmpl"),
 )

IMAGE_FOLDERS = {CONTENT_HOME + 'images': 'images'}

GALLERY_FOLDERS = {CONTENT_HOME + "galleries": "galleries"}

```
I haven't tested everything, but this seems to be on the correct track. But the gallery generation fails when doing this. 

I don't uses the gallery, so I could just disable the plugin for now. 

Question for people with an idea of the internals of the code: is there anything advising against continuing down that path, with potential hidden issues lurking? I could probably also get to what I want with some symlinking...

**Comment by ralsina** [comment, marked as answer]
It's just that nobody has tried it. If you like it that way, go ahead.

**Reply by nabajour** [reply]
Thanks, if they are no obvious showstoppers, I'll try and come bug people again if I run into trouble :-P

**Comment by KertLynx** [comment]
Just comment out the GALLERIES line. It works for me.
